nusight2/src/client/components/localisation/model.ts

```diff
@@ -125,7 +125,7 @@ export class LocalisationModel {
   @observable particlesVisible = true;
   @observable goalsVisible = true;
   @observable fieldLinePointsVisible = true;
-  @observable fieldIntersectionsVisible = true;
+  @observable fieldIntersectionsVisible = false;
```

Comment:
?

Reply:
Intersection associations show up when sending Field messages which we don't want by default